package.json

```diff
@@ -50,10 +50,17 @@
     "cy": "CYPRESS_baseUrl=http://localhost:9999 cypress open",
     "cy:dev": "source ../monitor-ci/.env && CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{cloud,shared}/**/*.*'",
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
-    "generate": "oats https://raw.githubusercontent.com/influxdata/influxdb/master/http/swagger.yml > ./src/client/generatedRoutes.ts && yarn unity && yarn flows && yarn managedFunctions",
-    "unity": "oats ./src/client/swagger/unity.yml > ./src/client/unityRoutes.ts",
-    "flows": "oats ./src/client/swagger/flows.yml > ./src/client/flowsRoutes.ts",
-    "managedFunctions": "oats ./src/client/swagger/managedFunctions.yml > ./src/client/managedFunctionsRoutes.ts"
+    "generate": "yarn currentApi && yarn unity && yarn flowd && yarn managedFunctions",
```

Comment:
will we want to update this to include running all the oats commands?

Reply:
maybe; likely sometime.

One of the things doing so would allow is to address the drift that has occurred between the oss and cloud apis. there is a `common` definition that continues to define the bits of the api that is 100% equal between the two platforms (not sure how those differences are handled in the ui currently, or if those drifted routes are even an issue)

For now, i don't want to add any disruption and only am generating ts clients from what should be identical swagger definitions to what is currently being consumed